src/hotel-management-app/src/app/restaurant/order-management/order-management.component.ts

```diff
@@ -3,16 +3,23 @@ import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { Location } from '@angular/common';
 import { CartService } from '../../services/cart.service';
+import { OrderStatus } from '../../models/statusCode.model';
 import { Order } from '../../models/orders.model';
 
+
 @Component({
   selector: 'app-order-management',
-  templateUrl: './order-management.component.html',
-  styleUrls: ['./order-management.component.css']
+  templateUrl: './order-management.component.html'
 })
 export class OrderManagementComponent implements OnInit {
   restaurantId!: number;
   orders: Order[] = [];
+  orderStatusOptions: OrderStatus = {
```

Comment:
Could this be type alone since the value isn't been used anywhere

Reply:
Resolved